README.md

```diff
@@ -148,21 +180,44 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
+> Write a message to encourage readers to support your project
+
+If you like this project...
```

Comment:
- [ ] Kindly complete the text. If someone likes this project what should the person do?